src/bundle/Command/MigrateRichTextNamespacesCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\FieldTypeRichText\Command;
+
+use Ibexa\FieldTypeRichText\Persistence\Legacy\MigrateRichTextNamespaces\Handler;
+use Symfony\Component\Cache\Adapter\TagAwareAdapterInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class MigrateRichTextNamespacesCommand extends Command
+{
+    protected static $defaultName = 'ibexa:migrate:richtext-namespaces';
+
+    private Handler $handler;
```

Comment:
Shouldn't `MigrateRichTextNamespacesHandlerInterface` be type-hinted here instead?